lib/download.go

```diff
@@ -1,47 +1,142 @@
 package lib
 
 import (
+	"errors"
 	"fmt"
 	"io"
+	"log"
 	"net/http"
 	"os"
 	"path/filepath"
 	"strings"
 )
 
-// DownloadFromURL : Downloads the binary from the source url
-func DownloadFromURL(installLocation string, url string) (string, error) {
+const (
+	pubKeyId     = "72D7468F"
```

Comment:
Would it make sense to allow users to customize the value?
E.g. if or when Hashicorp get the key rotated so that users don't depend on the new release of `tfswitch`?

Reply:
I think in this case we should cover that with a new release.

Reply:
Agree. My point is that we're better off not introducing features that may block users as this had happened e.g. when Hashi moved to another site to distribute releases — that block all of the `tfswitch` users.